src/clr-angular/data/datagrid/datagrid.ts

```diff
@@ -124,7 +124,7 @@ export class ClrDatagrid implements AfterContentInit, AfterViewInit, OnDestroy {
     /**
      * Selection/Deselection on row click mode
      */
-    @Input("clDgRowSelection")
+    @Input("clrDgRowSelection")
```

Comment:
I think we need to stay backwards compatible here. I believe that you can keep the new input and re-add the original ```@Input("clDgRowSelection")```

Reply:
@hippee-lee 
You're right. I've considered it in the implementation. I'll add it again.

Thank you for your reminder.

Reply:
@hippee-lee re-added, I am not sure whether the Implementation meets your coding rule.